pkg/backup/util/util.go

```diff
@@ -104,6 +135,7 @@ func GenerateStorageCertEnv(backup *v1alpha1.Backup, secretLister corelisters.Se
 	name := backup.GetName()
 
 	var certEnv []corev1.EnvVar
+	var reason string
```

Comment:
why not add reason into err msg ?

Reply:
The reason for the failure will be recorded in the backup's status. Currently, use string to represent it. I will define a type to describe it later.